### DISCUSSION TITLE
如何知道自己的電腦應下載32bit的執行檔，還是64bit的執行檔(2/2)

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#159] [Ideas]
之前分享過查看電腦作業系統的第一種方法，現在要分享第二種方法。
# 方法二
檔案總管->以滑鼠右鍵點擊「本機」一下->以滑鼠左鍵點選「內容(R)」一下->關於
## 第一步
以滑鼠左鍵點選「檔案總管」一下
<img width="960" alt="檔案總管" src="https://user-images.githubusercontent.com/86505454/142733072-3ef03f3a-018a-40e4-86a2-f1886890d063.png">

## 第二步
以滑鼠右鍵點選「本機」一下
<img width="960" alt="以滑鼠右鍵點擊「本機」一下" src="https://user-images.githubusercontent.com/86505454/142733105-dbb6756b-b3b3-4741-b63c-a91a1e85170e.png">

## 第三步
以滑鼠左鍵點選「內容」一下
<img width="960" alt="以滑鼠左鍵點選「內容(R)」一下" src="https://user-images.githubusercontent.com/86505454/142733133-2883af53-aa4c-44aa-baad-358e550a2732.png">

## 第四步
以滑鼠左鍵點選「關於」一下。就是上面的「關於。
<img width="960" alt="以滑鼠左鍵點選「關於」一下" src="https://user-images.githubusercontent.com/86505454/142733160-4edb8a81-e9b5-4157-a1c5-49a625d7eedf.png">

## 得知電腦作業系統類型
<img width="960" alt="電腦作業系統類型" src="https://user-images.githubusercontent.com/86505454/142733169-7284446b-fb3c-4c1d-9ab8-21b44451de51.png">
「64位元」就是「x64」就是「64bit」，因此此電腦為64bit的作業系統。

**Comment by enoch-lightning** [comment]
32bit的電腦作業系統，無法執行(還是安裝？)64bit的執行檔。64bit的電腦作業系統，可以執行(還是安裝？)32bit的執行檔，也能執行(還是安裝？)64bit的執行檔。

**Reply by chia-kai-chang** [reply]
執行喔

**Reply by enoch-lightning** [reply]
收到，謝謝老師。執行檔就是拿來執行的檔案，所以與安裝無關。
